content/pages/accelerate.md

```diff
@@ -0,0 +1,18 @@
++++
+title = "Verso ACCELERATE - Modular Abstraction Layer for Simplified Integration of Machine Learning Models"
```

Comment:
```suggestion
title = "ACCELERATE - Modular Abstraction Layer for Simplified Integration of Machine Learning Models"
```

I think it should be named as standalone project itself.

Reply:
I always felt that this was an integral part of what we were doing from a larger project perspective. Removing the "Verso" context is just optics, but I guess it aligns with the FORTRESS naming...